ts/Extensions/Breadcrumbs.ts

```diff
@@ -0,0 +1,1180 @@
+    /**
+     * Performs multiple drillups based on the provided number.
+     *
+     * @requires  modules/breadcrumbs
+     *
+     * @function Highcharts.Breadcrumbs#jumpBy
+     * @param {Highcharts.Breadcrumbs} this
+     *        Breadcrumbs class.
+     * @param {Highcharts.Breadcrumbs} jumpsAmount
+     *        A number of drillups that needs to be performed.
+     */
+    public jumpBy(this: Breadcrumbs, jumpsAmount: number): void {
+        const breadcrumbsList = this.breadcrumbsList;
+
+        // if defined, new level is a current level - jumps amount
+        // if not defined, new level is usually 0 - the first level in list.
+        if (breadcrumbsList.length) {
+            const jumpsNumber = defined(jumpsAmount) ?
+                breadcrumbsList[breadcrumbsList.length - 1][0] - jumpsAmount :
+                breadcrumbsList[breadcrumbsList.length - 1][0];
+
+            if (this.options.showFullPath) {
+                for (let i = 0; i < jumpsNumber; i++) {
+                    this.jumpUp();
+                }
+            } else {
+                this.jumpUp();
+            }
+            this.chart.getMargins();
+
+            if (!this.level) {
+                this.destroyGroup();
+                this.chart.redraw();
+            }
+        }
+    }
+
+    /**
+    * Redraw.
+    *
+    * @requires  modules/breadcrums
+    *
+    * @function Highcharts.Breadcrumbs#redraw
+    * @param {Highcharts.Breadcrumbs} this
+    *        Breadcrumbs class.
+    */
+    public redraw(this: Breadcrumbs): void {
+        if (this.isDirty) {
+            this.render();
+        }
+
+        if (this.breadcrumbsGroup && this.breadcrumbsList.length) {
+            this.breadcrumbsGroup.align();
+        }
+
+        this.isDirty = false;
+    }
+
+    /**
+    * Update list after the drillUp.
+    *
+    * @requires  modules/breadcrums
+    *
+    * @function Highcharts.Breadcrumbs#updateBreadcrumbsList
+    * @param {Highcharts.Breadcrumbs} this
+    *        Breadcrumbs class.
+    */
+    public updateBreadcrumbsList(this: Breadcrumbs): void {
+        const breadcrumbsList = this.breadcrumbsList;
+        if (this.level === 0) {
+            this.destroyGroup();
+        } else {
+            if (this.options.showFullPath) {
+                // last breadcrumb
+                const lastB = breadcrumbsList[breadcrumbsList.length - 1],
+                    button = lastB && lastB[2],
+                    conector = lastB && lastB[3],
+
+                    prevConector = breadcrumbsList[breadcrumbsList.length - 2] &&
+                        breadcrumbsList[breadcrumbsList.length - 2][3];
+
+                // Remove connector from the previous button.
+                prevConector && prevConector.destroy();
+                breadcrumbsList[breadcrumbsList.length - 2].length = 3;
+
+                // Remove SVG elements fromt the DOM.
+                button && button.destroy();
+                conector && conector.destroy();
```

Comment:
Duplicated code from `destroyGroup`.

Reply:
It is slightly different, as it removes only on button/connector. I have also removed `button.destroy` and `connector.destroy` from `destroyGroup` so it is not duplicate anymore